pkg/services/accesscontrol/models.go

```diff
@@ -186,3 +186,22 @@ func (p RoleDTO) Role() Role {
 		Created:     p.Created,
 	}
 }
+
+const (
+	// Permission actions
```

Comment:
There is a trade-off between declaring these as constants vs not. I think having them declared in one places is a huge plus for keeping it as a source of truth and documentation.

Reply:
Makes sense. I'm not a fond of globals and having to extend the mental lookup table to include a bunch of constants, but yeah, this might prevent more bugs than it'll introduce.